### ISSUE TITLE
[Bug]: Adding site admin dropdown is hard to use

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/PlaceCal#1756]
## Description

What were you doing?

![image](https://user-images.githubusercontent.com/861172/226913562-a0d684ce-4b4a-43b2-b967-67231abe34d0.png)

## Steps to reproduce

1. What you did first
1. What you did next

## What you expected to happen

Not 100% width & not have to type into secondary box

## Platform (if relevant)

What device and browser were you using?

**Comment by kimadactyl** [comment]
What about this did you find hard? Just the way it pops up the extra box?

**Comment by katjam** [comment]
> What about this did you find hard? Just the way it pops up the extra box?

Yes, clicking in to that second box is not intuitive and the width.
